articles/article-047-game-of-life.md

```diff
@@ -0,0 +1,27 @@
+<!-- BEGIN TITLE -->
+# Game of life
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+John Horton Conway invented a simple game with a small amount of rules. 
+Each cell on a square grid is either dead or alive. For each iteration the cell looks at it eight neighbors.
+If it has less than two living neighbors it dies of loneliness, but if it has more than three neighbors it dies of overpopulation. A cell turns alive if it has exactly three living neigbors.
+
+Those rules are quite simple but the possibilities of interesting simulation are enormous. There is a whole zoo of creatures and self replicating patterns that can evolve in this two dimensional world. The Game of life is Turing complete and simulating it is often used to show Turing completeness of something else.
```

Comment:
Those rules are quite simple but the possibilities, of interesting simulations, are enormous. 
- reason: I split the of since it just makes it clearer what is enormous. I also assume you can simulate multiple times.